src/applications/check-in/day-of/tests/e2e/confirmation-display/confirmation.display.cypress.spec.js

```diff
@@ -10,6 +10,10 @@ import Confirmation from '../pages/Confirmation';
 
 describe('Check In Experience -- ', () => {
   describe('Confirmation display -- ', () => {
+    const appointments = [
+      { startTime: '2021-08-19T03:00:00' },
+      { startTime: '2021-08-19T03:30:00' },
```

Comment:
The only thing a little confusing about this is the startTime, are these times getting overwritten in the createAppointment function? I don't see them getting passed into the function. It is kind of weird to define these appointments in the past If it is just to define the number of appointments. Maybe doing something like { eligibility: 'ELIGIBLE' } would make more sense when reading it.

Reply:
Ah, I see that the Object.assign gives precedent to the source. So this makes it so each appointment doesn't have the same startTime. Still kind of weird, but maybe out of the scope of this ticket to fix.